hathor/manager.py

```diff
@@ -1051,36 +1055,31 @@ def on_new_tx(self, tx: BaseTransaction, *, conn: Optional[HathorProtocol] = Non
             # This needs to be called right before the save because we were adding the children
             # in the tx parents even if the tx was invalid (failing the verifications above)
             # then I would have a children that was not in the storage
-            tx.update_initial_metadata(save=False)
+            # tx.update_initial_metadata(save=False)
```

Comment:
Should we remove these two commented lines? I feel we will get back to it soon in another PR.

Reply:
Done.

Reply:
Ok!